apps/quartz-app/src/helpers/chart.ts

```diff
@@ -0,0 +1,6 @@
+export const getDomainWithUpperBuffer = (
+  [dataMin, dataMax]: [number, number],
+  buffer: number
+): [number, number] => {
+  return [0, Math.floor((dataMax + buffer) / 1000) * 1000 + 1000];
+};
```

Comment:
1. Could you just take in dataMax and not dataMin, as you dont use dataMin
2. Couldnt you use `Math.ceil((dataMax + buffer)/1000) *1000), then you wouldnt have to add the extra 1000
3. could you make `1000` a variable`, say called `rounding_amount` so its more equally changed in the future
4. Can you add a function descirption, say what the function does and what the variables are

Reply:
<img width="655" alt="Screenshot 2024-04-30 at 09 46 35" src="https://github.com/openclimatefix/quartz-frontend/assets/34686298/9a0af0cc-9186-4ccd-8064-16eb3fb44a6d">
This seems the common way?

Reply:
1. This was a toss up as this function was being received by an external component that gives those params by default – have now wrapped the invocation to make the util function cleaner though as suggested;
2. Absolutely, not sure why I did it that way, probs an early train thing – have amended;
3. Done;
4. Done.